my_game.py

```diff
@@ -471,6 +478,18 @@ def on_show_view(self):
         # setup spawn_ufo to run regularly
         arcade.schedule(self.spawn_ufo, CONFIG['UFO_SPAWN_RATE'])
 
```

Comment:
It should not be called a _spinner_. Update so we know what this is for.

Reply:
It is not anymore! YEP YEAH UHUH LETS GOOOOOOOOOOOOOOOOOOOOOOOOOOOOO!